### PR TITLE
base64 on python3 bytes fix

### PR DESCRIPTION
**Pull request by mcXrd**

### CODE REVIEW

File: django_mojeid/models.py [hunk 1]
```diff
@@ -47,7 +48,7 @@ def __init__(self, *args, **kwargs):
 
         # Generate default salt
         if not 'salt' in kwargs:
-            kwargs['salt'] = os.urandom(30).encode('base64')[:-1]
+            kwargs['salt'] = base64.b64encode(os.urandom(30))
```
Comment (matllubos) [inline django_mojeid/models.py:51]: otestoval jsi to na produkci, mě z chyby připadá že jde úplně o něco jiného

Comment (mcXrd) [reply django_mojeid/models.py:51]: https://usertech.atlassian.net/browse/KNJ-3015

File "/srv/apps/KNJ/app/var/ve/lib/python3.5/site-packages/django_mojeid/models.py" in **init**
1.             kwargs['salt'] = os.urandom(30).encode('base64')[:-1]

Exception Type: AttributeError at /mojeid/callback/
Exception Value: 'bytes' object has no attribute 'encode'

Comment (mcXrd) [reply django_mojeid/models.py:51]: sel jsem podle tohoto tracebacku co byl v tasku, jak to mam otestovat na produkci netusim, opravil jsem proste tuhle chybu, jestli je tam i nejaka jina, o te nevim

Comment (matllubos) [reply django_mojeid/models.py:51]: ještě je tam tato chyba

File "/srv/apps/KNJ/app/var/ve/lib/python3.5/site-packages/django/core/handlers/exception.py" in inner
1.             response = get_response(request)

File "/srv/apps/KNJ/app/var/ve/lib/python3.5/site-packages/django/core/handlers/base.py" in _legacy_get_response
1.             response = self._get_response(request)

File "/srv/apps/KNJ/app/var/ve/lib/python3.5/site-packages/django/core/handlers/base.py" in _get_response
1.                 response = self.process_exception_by_middleware(e, request)

File "/srv/apps/KNJ/app/var/ve/lib/python3.5/site-packages/django/core/handlers/base.py" in _get_response
1.                 response = wrapped_callback(request, _callback_args, *_callback_kwargs)

File "/srv/apps/KNJ/app/var/ve/lib/python3.5/site-packages/django/views/generic/base.py" in view
1.             return self.dispatch(request, _args, *_kwargs)

File "/srv/apps/KNJ/app/var/ve/lib/python3.5/site-packages/django/views/generic/base.py" in dispatch
1.         return handler(request, _args, *_kwargs)

File "/srv/apps/KNJ/app/var/ve/lib/python3.5/site-packages/django_mojeid/views.py" in get
1.             context_instance=RequestContext(request)

Exception Type: TypeError at /mojeid/
Exception Value: render_to_response() got an unexpected keyword argument 'context_instance'
Request information:
USER: AnonymousUser

**Pull request merged**